test/clj/puppetlabs/trapperkeeper/services/webserver/normalized_uri_helpers_test.clj

```diff
@@ -116,8 +116,8 @@
     ;;> or the surrogate pair ED A1 8C ED BE B4 into U+233B4.  Decoding
     ;;> invalid sequences may have security consequences or cause other
     ;;> problems.
-    (is (= "��" (normalize-uri-path-for-string "%C0%AE")))
-    (is (= "/foo/��/��" (normalize-uri-path-for-string "/foo/%C0%AE/%C0%AE")))))
+    (is (= "-64-82" (normalize-uri-path-for-string "%C0%AE")))
```

Comment:
Hmm, this seems unfortunate.  Do you know why this behavior changed?  Ideally, it would not have.